src/camera/Camera.h

```diff
@@ -0,0 +1,295 @@
+#pragma once
+
+#include <memory>
+#include <mutex>
+#include <string>
+#include <thread>
+
+#include <opencv2/core.hpp>
+#include <opencv2/videoio.hpp>
+
+#include "CameraParams.h"
+
+/**
+   @defgroup camera Camera Access
+   @brief This module provides an interface for defining and accessing the
+   different cameras attached to the rover.
+
+   Cameras should be defined in a YAML file with a format that will be
+   documented soon, and the location of this file should be able to be
```

Comment:
Looks like this was already documented?

Reply:
Woops, good catch. I wrote that way before writing the config file format docs and forgot to change it.

Reply:
Fixed in 759bbaa2e85f16fe33c7aff486c34636fcb857bd.